AI-and-Analytics/Getting-Started-Samples/LPOT-Sample-for-Tensorflow/sample.json

```diff
@@ -16,7 +16,7 @@
 			"conda create --name user_tensorflow -y",
 			"conda activate user_tensorflow",
 			"conda install -c ${ONEAPI_ROOT}/conda_channel tensorflow numpy pyyaml scikit-learn schema lpot ilit matplotlib -y",
```

Comment:
might be better to install in user_tensorflow conda env by "-n user_tensorflow", and might be better to move the installation into "steps" section.

Reply:
-n and --name are equal in fact:

Target Environment Specification:
  -n ENVIRONMENT, --name ENVIRONMENT
                        Name of environment.

The "steps" should include the test steps, instead of prepare running environment.
So, I think keep conda install cmds in "env" section is better.